src/CMakeLists.txt

```diff
@@ -377,6 +377,11 @@ target_compile_definitions(Halide
                            $<$<STREQUAL:$<TARGET_PROPERTY:TYPE>,STATIC_LIBRARY>:Halide_STATIC_DEFINE>
                            $<$<CXX_COMPILER_ID:GNU,Clang,AppleClang>:WITH_INTROSPECTION>)
 
+include(TargetExportScript)
+target_export_script(Halide
+                     APPLE_LD "${CMAKE_CURRENT_LIST_DIR}/exported_symbols.osx"
+                     GNU_LD "${CMAKE_CURRENT_LIST_DIR}/exported_symbols.linux")
+
```

Comment:
Redundant, but let's include

```
    ## TODO: implement something similar for Windows/link.exe
    # https://github.com/halide/Halide/issues/4651
```

here too, so that a reader doesn't wonder if special casing for windows was omitted accidentally or not

Reply:
Looks like you already did that 🙂